imagedephi/redact/redact.py

```diff
@@ -17,8 +17,16 @@
 from .tiff import UnsupportedFileTypeError
 
 
-def _get_output_path(file_path: Path, output_dir: Path) -> Path:
-    return output_dir / f"REDACTED_{file_path.name}"
+def _get_output_path(
+    file_path: Path,
+    output_dir: Path,
+    base_name: str,
+    count: int,
+    max: int,
+) -> Path:
+    number_length = len(str(max))
+    number = str(count).zfill(number_length)
+    return output_dir / f"{base_name}_{number}{file_path.suffix}"
```

Comment:
I think you can just do `return output_dir / f"{base_name}_{number:0{len(str(max))}d}{file_path.suffix}"`

Reply:
And that `d` might not be needed since `number` is an integer.

Reply:
Thanks. This has been added in bdb5707